src/metabase/logger.clj

```diff
@@ -22,7 +23,7 @@
    :level        (.getLevel event)
    :fqns         (.getLoggerName event)
    :msg          (.getMessage event)
-   :exception    (.getThrowableStrRep event)
+   :exception    (str/join "\n" (.getThrowableStrRep event))
```

Comment:
I think it might be better to do this on the UI side, treating the formatting as a UI concern vs a logger concern.

also, this will end up with exception set in all cases, which isn't ideal. it should be a nil where there is no corresponding stacktrace.

```
(clojure.string/join "\n" nil)
""
```

Reply:
Agreed. I joined the strings here because the returned array wasn't serializing correctly. (see PR message) Any thoughts on how to avoid that?

Reply:
oh, I see why you did that. `"[Ljava.lang.String;@3c78dc82"` is the `toString` representation of an array in Java. Sending a string array should be serializable to JSON without any issues. That bug is elsewhere, it looks like.